### PR TITLE
Only mark 2.1.x releases as the default

### PR DESCRIPTION
**Pull request by wilkinsona**

### CODE REVIEW

**Comment by vpavic** [comment]
I've landed here via spring-projects/spring-boot#12835 - it would be nice if this was addressed since with the release of Spring Boot `1.5.18.RELEASE` SDKMAN has again started to offer it as upgrade over 2.x:

```bash
$ sdk ug
==== BROADCAST =================================================================
* 29/11/18: Springboot 1.5.18.RELEASE released on SDKMAN! #springboot
* 26/11/18: Gradle 5.0 released on SDKMAN! #gradle
* 22/11/18: Gradle 5.0-rc-5 released on SDKMAN! #gradle
================================================================================

Upgrade:
springboot (2.1.0.RELEASE < 1.5.18.RELEASE)

Upgrade candidate(s) and set latest version(s) as default? (Y/n):
```

File: gvmpublish.groovy [hunk 1]
```diff
@@ -39,7 +39,7 @@ class GvmPublish implements ApplicationRunner {
 			.body('{"candidate": "springboot", "version": "' + version + '", "url": "' + downloadUrl + '"}')
 		rest.exchange(releaseRequest, String.class)
 
-		if (version.endsWith("RELEASE")) {
+		if (version.startsWith("2.0.") && version.endsWith("RELEASE")) {
```
Comment (vpavic) [inline gvmpublish.groovy]: With Spring Boot 2.1 out, this line should probably be updated.

**Comment by wilkinsona** [comment]
Thanks, @vpavic. I've force-pushed this branch with some changes I had locally. I've also marked 2.1.0 as the default again.

**Comment by vpavic** [comment]
> I've also marked 2.1.0 as the default again.

Thanks - I can confirm it doesn't offer upgrade anymore.

**Pull request merged**